### ISSUE TITLE
Learning Sound Event Classifiers from Web Audio with Noisy Labels

### ISSUE THREAD

**Issue by YoshikiMas** [YoshikiMas/Survey-ICASSP2019#5]
## 0. タイトル
Learning Sound Event Classifiers from Web Audio with Noisy Labels

## 1. ひとこと
Youtubeなどweb上で収集したノイズラベルの多い条件での音響イベント検出．

## 2. 先行研究との比較
ラベルノイズの影響はCVや一般の機械学習では盛んに研究されているが，音響ではあまり注目されてこなかった．

## 3. 手法のキモ
FSDnoisy18kというラベルの正解/不正解やどのように不正解かを分類したデータセットを公開．
画像認識で提案されたラベルノイズにロバストなロスを比較．

## 4. 実験方法
FSDnoisy18kでベースライン(categorical crossentropy)と複数のロスを比較．
ラベルノイズにロバストなロスが有効であることを確認．

## 5. 議論
適切なロスを使うことで，手動でラベルの整理をするのと比較し非常に容易に性能向上が可能

## 6. 次に読むべき論文
[Training
deep neural networks on noisy labels with bootstrapping](https://ai.google/research/pubs/pub43273)
[Generalized cross entropy loss for training deep neural networks with noisy labels](http://papers.nips.cc/paper/8094-generalized-cross-entropy-loss-for-training-deep-neural-networks-with-noisy-labels)
